### ISSUE TITLE
[Feature Suggestion] Scrollbar for track list

### ISSUE THREAD

**Issue by metronidazole** [Taiko2k/TauonMusicBox#453]
It would be nice to have a scrollbar for the track list.

**Comment by Taiko2k** [comment]
For the main centre tracklist? There is one already, its just in an unorthodox position; hidden to the left side of the tracklist.

**Comment by metronidazole** [comment]
Oh there it is. I've never seen a left-sided scrollbar in my decades of experience. 

Can we add a setting to control which side of the track list the scrollbar will be positioned?

**Comment by Taiko2k** [comment]
> Can we add a setting to control which side of the track list the scrollbar will be positioned?

For better for worse, Tauon and the visual balancing of the tracklist is designed around the scroll bar being on the left.

Bluntly adding such a setting is possible. But adding the setting would be work just to downgrade the experience as the way I see it. I worry that users may see that setting and think "Oh scroll bars are *supposed* to be on the right hand side and its what I'm used to so I better toggle that", when I see it as preferable that the user learn and accept the left hand scroll bar.

**Comment by metronidazole** [comment]
Isn't that just a Steve Jobs argument - "you're holding it wrong"?

**Comment by trivigy** [comment]
@Taiko2k just wanted to add my two cents here. I found it incredibly weird that the scroll is on the left. At first I just thought there was no scrollbar at all. Once I got the gist of the settings and setup a nice looking minimalist look I noticed the scrollbar.

> But adding the setting would be work just to downgrade the experience as the way I see it.

As you are pointing it out yourself, preference is a subjective opinion. So it is hard to argue. Especially when the result of the decision requires development work. However two analytical counter argument that I would like to add are:

- It requires a longer mouse drag when navigating from other windows scroll bars found on windows in the background. Being that the scrollbars are on opposite sides of the screen.
- A lefty/righty user will have an easier time associating scrollbar side with where the mouse is found in relations to the computer.

**Comment by Taiko2k** [comment]
Yeah, I agree its bad, even I myself keep subconsciously trying to find on the right. But its not a simple fix because it doesn't *fit* as is on the right without making visual changes (And it would intersect with the text if overlayed).

**Comment by IfGremlinThen** [comment]
The scrollbar on the left is basically the only thing I don't like about Tauon (other than the formats it's available in).

There's at least 6 layout options and all these ways to theme the player, but you can't move the scrollbar to right like every other app?  It's just weird.

**Comment by Solarunit** [comment]
I would like to have an option to put scrollbar to the right side too in the track-list. Maybe even for playlist list in the left panel too.
